subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/impl/AbsolutePathFileCollectionFingerprinterTest.groovy

```diff
@@ -77,7 +77,7 @@ class AbsolutePathFileCollectionFingerprinterTest extends Specification {
         def fingerprint = fingerprinter.fingerprint(files(file, file2, file3, file4))
 
         then:
-        fingerprint.fingerprints.keySet().collect { new File(it) } == [file, file2, file3, file4]
+        fingerprint.fingerprints.keySet().collect { new File(it) } == [file, file4]
```

Comment:
For this test `file2` and `file3` should be created, not removed from the test.

Reply:
I changed it to create directories instead.